frontend/src/pages/auth/components/fields/password/props.ts

```diff
@@ -0,0 +1,4 @@
+export interface PasswordUserCredentialsProps {
+    defaultPassword: string;
```

Comment:
we never return password from server, so it's not possible to provide default password. User has to type it each time from the scratch